### PR TITLE
fix(Video, DisplayMediaCard, PosterCard): show fallback on empty src

### PR DESCRIPTION
**Pull request by marcoskolodny**
Issue: [Link](https://jira.tid.es/browse/WEB-1557)

### CODE REVIEW

File: src/card.tsx [hunk 1]
```diff
@@ -993,21 +991,25 @@ const DisplayCard = React.forwardRef<HTMLDivElement, GenericDisplayCardProps>(
         },
         ref
     ) => {
+        const hasImage = backgroundImage !== undefined;
+        const hasVideo = backgroundVideo !== undefined;
         const image = renderBackgroundImage(backgroundImage);
         const {video, videoAction} = useVideoWithControls(backgroundVideo, poster, backgroundVideoRef);
 
-        if (backgroundVideo) {
+        console.log(hasImage, hasVideo);
```
Comment (Winde) [inline src/card.tsx:999]: probably needs to be removed?

**Review by Winde — APPROVED** [review]

**Review by atabel — APPROVED** [review]
I miss a screenshot test with this case

**Review by pladaria — APPROVED** [review]
you could update the test case where an invalid URL is provided to force the fallback to use an empty string (or add a new case to verify the empty string behavior)

https://github.com/Telefonica/mistica-web/blob/45d07bfdc7798e2beebef3700ea014772402e774/src/__stories__/image-story.tsx#L72

**Pull request merged**